### ISSUE TITLE
Fail to install Midas 3.2.20

### ISSUE THREAD

**Issue by felixveysseyre** [midasplatform/Midas#259]
Hello,

I have been trying to install Midas 3.2.20 (on Mac OS X 10.11.2) but I am stuck on step 2.

After entering my database information correctly, I got the following error message:

> Could not find mixer
> In /Users/felixveysseyre/Projects/Midas/vendor/ircmaxell/random-lib/lib/RandomLib/Factory.php, line: 187
> At 06:09:37 2016-09-02

I have tried to update Random-lib package but I am still getting the same error.

<img width="616" alt="capture d ecran 2016-09-02 a 12 09 53" src="https://cloud.githubusercontent.com/assets/4909259/18200732/912c4840-7106-11e6-90b5-517e52b182b6.png">

Thanks,
Félix

**Comment by cpatrick** [comment]
I believe this is an issue with not finding _some_ underlying crypto library, do you have  the openssl or mcrypt extension installed?

At any rate, we should improve the error message.

**Comment by felixveysseyre** [comment]
_OpenSSL_ extension is installed, not _mcrypt_.

<img width="639" alt="capture d ecran 2016-09-02 a 14 10 26" src="https://cloud.githubusercontent.com/assets/4909259/18203489/93b50b54-7117-11e6-9ada-7e858b2ff312.png">


Is _mcrypt_ a dependency ? I did not get any error message on step1.

**Comment by cpatrick** [comment]
it shouldn't be, does installing it address the issue?

**Comment by felixveysseyre** [comment]
Same error with _mcrypt_ installed:
<img width="638" alt="capture d ecran 2016-09-02 a 14 28 35" src="https://cloud.githubusercontent.com/assets/4909259/18203850/cbeddfee-7119-11e6-837d-8e05791b4554.png">

**Comment by cpatrick** [comment]
Hmmm, Are you able to manually create a RandomLib Matcher?

@jamiesnape, have you seen anything like this?

**Comment by jamiesnape** [comment]
No, but we do not test old versions of Midas with recent dependencies. Is there a compelling reason to be using 3.2.20? It is missing critical security fixes.

**Comment by felixveysseyre** [comment]
@jamiesnape Mostly because our old applications (two or three years old) have been implemented as private module working with Midas 3.2.

It would make sens to use 3.4 instead if it was not requiring an important amount of work.

**Comment by felixveysseyre** [comment]
@cpatrick Disabling _openssl_ extension allowed me to complete the installation.

```
if (false && extension_loaded('openssl')) {
    $factory = new \RandomLib\Factory();
    $generator = $factory->getHighStrengthGenerator();
    $prefix = $generator->generateString(32);
} else {
    $prefix = $this->Component->Random->generateString(32);
}
```

I am not sure what is happening but it could be related to this issue https://github.com/ircmaxell/RandomLib/issues/25

Thanks @cpatrick !
